plugin/pkg/scheduler/factory/factory.go

```diff
@@ -131,13 +131,7 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys util.StringSe
 
 	// Watch minions.
 	// Minions may be listed frequently, so provide a local up-to-date cache.
-	if false {
```

Comment:
@ddysher 

You are right. I saw your recent PR also where you are expecting kubelet to post Node status. Is kubelet posting to apiserver so its available in the scheduler also?